### PR TITLE
Add Seller Search

### PR DESCRIPTION
**Pull request by gustavopvasconcellos**
#### What problem is this solving?

Facets doesn't work with different `map` props and we have a necessity to add a seller page, to make this possible, we created a Wrapper to the Search Context and sends to it the term as a query and "seller" as a map.

#### How to test it?

[Workspace](https://seller--carrefourbr.myvtex.com/parceiro/homepoint)

#### Describe alternatives you've considered, if any.

I've thought about change the mainSearches method to make more generic, but this will need a plan to don't break other sites...

### CODE REVIEW

File: react/hooks/useFacetNavigation.js [hunk 1]
```diff
@@ -158,7 +159,11 @@ export const buildNewQueryMap = (
   let querySegments = selectedFacets.map(facet => facet.value)
   let mapSegments = selectedFacets.map(facet => facet.map)
 
-  const { ft: fullText, productClusterIds: collection } = fullTextAndCollection
+  const {
+    ft: fullText,
+    productClusterIds: collection,
+    seller,
+  } = fullTextAndCollection
```
Comment (iaronaraujo) [inline react/hooks/useFacetNavigation.js:166]:
```suggestion
  } = fullTextSellerAndCollection
```

**Review by iaronaraujo — CHANGES_REQUESTED** [review]
Code looks good! Just left a small suggestion!

**Review by iaronaraujo — APPROVED** [review]

**Comment by lucis** [comment]
@gustavopvasconcellos update the tests 
image

File: react/hooks/useFacetNavigation.js [hunk 2]
```diff
@@ -158,7 +159,11 @@ export const buildNewQueryMap = (
   let querySegments = selectedFacets.map(facet => facet.value)
   let mapSegments = selectedFacets.map(facet => facet.map)
 
-  const { ft: fullText, productClusterIds: collection } = fullTextAndCollection
+  const {
+    ft: fullText,
+    productClusterIds: collection,
+    seller,
+  } = fullTextSellerAndCollection
```
Comment (hiagolcm) [inline react/hooks/useFacetNavigation.js:166]: There is no `fullTextSellerAndCollection ` declared in the code. It is throwing an error

image

**Review by hiagolcm — CHANGES_REQUESTED** [review]

**Pull request merged**